package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-openshift-templates",
   "description": "OpenShift V3 Templates for FH MBaaS",
-  "version": "0.24.87",
+  "version": "0.24.88",
```

Comment:
I'd suggest a minor bump here, since you've added new stuff. So like `0.25.0` I guess. What do you think? Also, there's a `templateVersion` annotation that should be kept in sync with this. I think if you run `npm version minor`, it will get everything (there's a custom script)